flake8 issues:
```
contract_compassion/models/contracts.py:235:25: E126 continuation line over-indented for hanging indent
contract_compassion/models/contracts.py:244:29: E126 continuation line over-indented for hanging indent
contract_compassion/models/contracts.py:250:5: E303 too many blank lines (2)
```

--- a/contract_compassion/models/contracts.py
+++ b/contract_compassion/models/contracts.py
@@ -232,20 +232,19 @@ class RecurringContract(models.Model):
             try:
                 # Try to get days difference between today and last letter
                 contract.last_letter = (
-                        date.today() - fields.Datetime.from_string(
-                            contract.env['correspondence'].search([
-                                ('direction', '=', 'Supporter To Beneficiary'),
-                                ('sponsorship_id', '=', contract.id)],
-                                limit=1).scanned_date).date()).days
+                    date.today() - fields.Datetime.from_string(
+                        contract.env['correspondence'].search([
+                            ('direction', '=', 'Supporter To Beneficiary'),
+                            ('sponsorship_id', '=', contract.id)],
+                            limit=1).scanned_date).date()).days
             except AttributeError:
                 try:
                     # Try to get days difference between today and create_date
                     contract.last_letter = (
-                            date.today() - fields.Datetime.from_string(
-                                contract.create_date).date()).days
+                        date.today() - fields.Datetime.from_string(
+                            contract.create_date).date()).days
                 except AttributeError:
                     contract.last_letter = 0
-
 
     ##########################################################################
     #                              ORM METHODS                               #

